### ISSUE TITLE
Param typo in GroupMilestones.edit()

### ISSUE THREAD

**Issue by mbrammer** [jdalrymple/gitbeaker#3307]
Hey,

looks like there is a typo in a parameter for `GroupMilestones.edit()` in the latest version of `@gitbeaker/rest: 39.0.0`. Regarding to the Gitlab API, your param `startEvent` should be named `stateEvent` ;)

<img width="917" alt="image" src="https://github.com/jdalrymple/gitbeaker/assets/2786547/f76196ea-24f2-4da1-bde6-43eb2b350f91">

**Comment by mbrammer** [comment]
Also looks like the param is being passed wrong to the Gitlab API.  
I tried it with `startEvent: 'close'` for now. but this resulted in the following error:
```
Error: Bad Request
    at throwFailedRequestError (/Users/mbrammer/Projects/vida/gitlab-slack/node_modules/@gitbeaker/rest/dist/index.js:78:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at defaultRequestHandler (/Users/mbrammer/Projects/vida/gitlab-slack/node_modules/@gitbeaker/rest/dist/index.js:110:7)
    at /Users/mbrammer/Projects/vida/gitlab-slack/node_modules/@gitbeaker/core/dist/index.js:152:22
    at /Users/mbrammer/Projects/vida/gitlab-slack/dist/src/gitlab/triage/triage.controller.js:44:21 {
  [cause]: {
    description: '"title, description, start_date, due_date, state_event are missing, at least one parameter must be provided"',
    response: Response {
      [Symbol(realm)]: null,
      [Symbol(state)]: [Object],
      [Symbol(headers)]: [HeadersList]
    }
  }
}
```

**Comment by jdalrymple** [comment]
Oops! Will fix asap!

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`39.1.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/39.1.0) :rocket:

**Comment by mbrammer** [comment]
Awesome! Thank you 🙏
